integration-test/common/conftest.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+import os
+import tempfile
+import shutil
+import subprocess
+
+RUN_SCRIPT_FNAME = "run.sh"
+OUTPUT_DIR = "output_dir"
+SWEEP_GENERATOR = "generate_design_sweeps.py"
+
+def is_master(config):
+  """Return true we're on a master node."""
+  return not hasattr(config, 'workerinput')
+
+def pytest_addoption(parser):
+  parser.addoption("--sweep-file", action="store", help="Sweep file")
+
+def pytest_configure(config):
+  """Generate sweeps using the sweep generator.
+
+  Only the master node performs the sweep generation.
```

Comment:
This is Python, so documentation on the args (especially what their types are) is essential.

Reply:
Since this is an API, I think documenting the arguments is not necessary since they have been better documented. But right, the specific use of this hook implementation should be explained.